Packs/Illumio/Integrations/IllumioCore/IllumioCore.py

```diff
@@ -198,15 +198,11 @@ def prepare_service_binding_output(response: dict) -> str:
     hr_outputs = []
 
     if response.get("errors"):
-        title = "Service Binding:\n#### Workloads have been already binded to the virtual service."
-        hr_outputs.append(
-            {"Status": response.get("errors", {"status": "uniqueness_failure"})[0].get("status")}
-        )
-        return tableToMarkdown(title, hr_outputs, headers=["Status"], removeNull=True)
+        raise ValueError("Workloads have been already binded to the virtual service.")
```

Comment:
This doesn't fix the problem. If a `/service_bindings` POST is made with multiple workload HREFs, some can succeed, and some can fail. If there's only one duplicate, but the POST itself returned a 200, we shouldn't raise an error here.

Reply:
So, can we do something like this, where we create a table for the successful workloads bindings & add a comment below the table stating how many workloads were already bound to the service, instead of raising the `ValueError`

Reply:
I've made the changes as per our discussion in the call.